spec/state_dmv_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
+RSpec.describe DmvFactory do
+    before(:each) do
+        @dmv_factory = DmvFactory.new
+    end
+
+    it 'maps facility to new array' do
+        @co_dmv_office_locations = DmvDataService.new.co_dmv_office_locations
+        facilities = @dmv_factory.create_co_facilities(@co_dmv_office_locations)
+        expect(facilities).to all(be_a(Facility))
+        expect(facilities).to be_an(Array)        
+    end
+
+    it 'Maps Colorado Data to new CO Objects' do
+        @co_dmv_office_locations = DmvDataService.new.co_dmv_office_locations
+        facilities = @dmv_factory.create_co_facilities(@co_dmv_office_locations)
+
+        expect(facilities.first.name).to eq("DMV Tremont Branch")
+        expect(facilities.dmv_id).to eq(1)
+    end
+    
+    
+    
+end
```

Comment:
No vehicle tests. Every method return value and side effect needs to be tested!! Try to do TDD so you are testing as you go!! Need to see an improvement in testing on your next projects.